doc/user/config.toml

```diff
@@ -135,6 +135,12 @@ parent = "integrations"
 url = "/integrations"
 weight = 5
 
+[[menu.main]]
+identifier = "cli-reference"
+name = "CLI command reference"
```

Comment:
Is `CLI command reference` under `Tools and Integrations` and not `CLI` because Hugo doesn't support indexing more than two consecutive children or is it intentional?

Reply:
> because Hugo doesn't support indexing more than two consecutive children

This! I'd love to have it nested, but I can't figure out how to do it with our current template.